app/controllers/settings/index.js

```diff
@@ -13,7 +13,7 @@ export default Controller.extend({
   notify: service(),
   lastUsed: storageFor('last-used'),
   user: alias('session.account'),
-  languages: getter(() => [{ id: 'en', text: 'English' }]),
+  languages: getter(() => [{ id: 'en', text: 'English' }, { id: 'fr', text: 'French' }]),
```

Comment:
Should be Français

Reply:
Yeah, mentionned it in the mail sent to Josh, would probably be better to localize languages, on the current one. But changing to "Français" for now.